tests/forms_tests/widget_tests/test_radioselect.py

```diff
@@ -2,30 +2,434 @@
 
 from django.forms import ChoiceField, Form, MultiWidget, RadioSelect
 from django.test import override_settings
+from django.utils.safestring import mark_safe
 
 from .base import WidgetTest
 
 
 class RadioSelectTest(WidgetTest):
     widget = RadioSelect
+    nested_widget = RadioSelect(
+        choices=(
+            ("outer1", "Outer 1"),
+            ('Group "1"', (("inner1", "Inner 1"), ("inner2", "Inner 2"))),
+        )
+    )
 
     def test_render(self):
         choices = (("", "------"),) + self.beatles
         self.check_html(
             self.widget(choices=choices),
             "beatle",
             "J",
-            html="""
+            html=(
+                """
+                <div>
+                  <div>
+                    <label><input type="radio" name="beatle" value="">------</label>
+                  </div>
+                  <div>
+                    <label>
+                    <input checked type="radio" name="beatle" value="J">John</label>
+                  </div>
+                  <div>
+                    <label><input type="radio" name="beatle" value="P">Paul</label>
+                  </div>
+                  <div>
+                    <label>
+                    <input type="radio" name="beatle" value="G">George
+                    </label>
+                  </div>
+                  <div>
+                    <label>
+                    <input type="radio" name="beatle" value="R">Ringo</label>
+                  </div>
+                </div>
+        """
+            ),
+        )
```

Comment:
Hey @nessita, I'm not sure why, but some tests weren't passing because they required parentheses

Reply:
I've re-worked this so we, consistently, define an `html` var to use in the assertion. This way, we have more horizontal space available and parenthesis are not needed.